@ABLITZER/findFishByKeyValuePairs.m

```diff
@@ -1,21 +1,27 @@
 
-% Find desired fish by providing tag-value pairs
-function indices = findFishByTagValuePairs(obj,varargin)
+% Find desired fish by providing key-value pairs
+
+%SYNTAX:
+%       1.  findFishByKeyValuePairs(obj,key1,value1,...)
+%       2. obj. findFishByKeyValuePairs(key1,value2,...)
+
+
+function indices = findFishByKeyValuePairs(obj,varargin)
```

Comment:
Maybe "findFishByKeys" better?

Reply:
Here we need to input key-value pairs , "KeyValuePairs " maybe is easier to understand ？